patches/api/0085-Add-PlayerAdvancementCriterionGrantEvent.patch

```diff
@@ -58,6 +64,18 @@ index 0000000000000000000000000000000000000000..bb8d7c959cdea4b66455a49e74804ea4
 +        return cancel;
 +    }
 +
++    /**
++     * Gets the AdvancementProgress.
++     * Unlike other methods, it returns the state after the event has completed.
++     * If the event is canceled, the change is rolled back after the event finishes.
```

Comment:
I think a better way would be something like this: Gets the AdvancementProgress with the involved criterion marked as granted. Would love to hear feedback on these suggestions though

Reply:
I think I might be confused that it implies only this advancement progress has it granted: other sources would still have it not marked, which is incorrect

maybe
gets the current AdvancementProgress after the criterion has been granted.

Reply:
i am unsure about this one. "after the criterion has been granted" kinda makes it seem like the criterion will get granted regardless of if the event is cancelled etc. perhaps we could add something like "See {@link PlayerAdvancementCriterionGrantEvent}" to my suggestion. since we already clarified it nicely in the class jdocs

Reply:
i think including the word current helps a lot, but i agree that linking back to original event is useful. is that compromise ok?

Reply:
yeah that seems nice